src/main/java/org/folio/spring/liquibase/FolioPostgresDatabase.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.spring.liquibase;
+
+import liquibase.database.core.PostgresDatabase;
+import liquibase.exception.DatabaseException;
+
+/**
+ * Override default behaviour that change search_path was set in {@link org.folio.spring.config.DataSourceFolioWrapper}
+ */
+public class FolioPostgresDatabase extends PostgresDatabase {
```

Comment:
Could you please elaborate more on the issue you've faced with?
In particular why only these methods are overriden

Reply:
@dmtkachenko, I have added this to the PR description.

Reply:
thanks, now it's more clear